prov/rxd/src/rxd.h

```diff
@@ -59,16 +59,17 @@
 #ifndef _RXD_H_
 #define _RXD_H_
 
-#define RXD_PROTOCOL_VERSION 	(2)
+#define RXD_PROTOCOL_VERSION	(2)
 
 #define RXD_MAX_MTU_SIZE	4096
 
-#define RXD_MAX_TX_BITS 	10
-#define RXD_MAX_RX_BITS 	10
+#define RXD_MAX_TX_BITS		10
+#define RXD_MAX_RX_BITS		10
```

Comment:
Just as an example, On vim these lines look exactly the same 1 tab distance. how to make such a change. also what do you mean by drop that which don't need to be fixed? can you point out what you are referring to?

Reply:
While fixing indentation. I also removed spaces in many parts of the code and replaced them with tabs, since there should be no spaces only tabs.  I had some confusion when a single line of code spans multiple lines. The lines after the first in some parts of the code were formatted differently, in different parts of the code, so I tried to make them uniform (for ex all second lines which are not function headers in a file to have the same number of tabs in a file from the left margin if it is not a RHS of an expression). Is this what you are referring to as changes which are not need to be fixed?

Reply:
See comment below. The spacing here is also ok. This one is here so that all the define values line up since RXD_MAX_MTU_SIZE is an extra character and requires an extra tab for the values below but it shows up as a space instead of a tab.

Reply:
Sorry just read your first comment here - if they look the same and it was just a space before and now a tab, this is an ok fix. Thanks!